### ISSUE TITLE
Error on upload

### ISSUE THREAD

**Issue by 1stevengrant** [4rn0/statamic-v2-image-optimizer#1]
Been getting this error when we upload into the control panel:

`getimagesize(/Users/steven/Sites/xxx/public/public/assets/nesa-by-makers-752736-unsplash-1-1537525996.jpg): failed to open stream: No such file or directory`

Thought it might be to do with uploading to an S3 container but the same error happens when the asset container is local.

The asset however does actually upload.

**Comment by jdsimcoe** [comment]
I'm seeing the same thing on a site that utilizes the plugin.

**Comment by einkoro** [comment]
This appears to occur on any install where you are running outside of the web root and move the assets container. Note the duplication of public here: .../public/public/assets...

You can see this by modifying the path in site/content/assets/main.yaml ... its fine with the default:
```
title: Main Assets
path: assets
url: /assets
```
But once you've moved it with the following the error occurs:
```
title: Main Assets
path: public/assets
url: /assets
```

**Comment by 4rn0** [comment]
Fixed by #2
